jme3-templates/src/com/jme3/gde/templates/basic/BasicGamePanelVisual.java

```diff
@@ -110,7 +110,7 @@ private void browseButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-
         String command = evt.getActionCommand();
         if ("BROWSE".equals(command)) {
             JFileChooser chooser = new JFileChooser();
-            FileUtil.preventFileChooserSymlinkTraversal(chooser, null);
```

Comment:
I am not sure about that, why has it been removed and what could be a replacement?